frontend/tests/ui/conftest.py

```diff
@@ -81,7 +81,7 @@ def _get_last_console_logs(self):
             yield entry
 
 
-class Driver:  # pylint: disable=too-many-instance-attributes
+class Driver:  # pylint: disable=too-many-instance-attributes, disable=R0904
```

Comment:
```suggestion
class Driver:  # pylint: disable=too-many-instance-attributes
```

qué pasa si se elimina? en general hay que evitar agregar disables de antemano, hay que discutir cómo se pueden resolver acá y si no se puede, pues se agrega. y de preferencia hay que usar el nombre largo en vez del código.

Reply:
Ok, este disable es porque se queja de que hay muchos métodos públicos: 21/20, qué podría hacer?

Reply:
```suggestion
class Driver:  # pylint: disable=too-many-instance-attributes, disable=too-many-public-methods
```